packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -352,3 +353,27 @@ class ImagePicker {
     return platform.getLostData();
   }
 }
+
+/// Android specific settings for [ImagePicker].
+class AndroidImagePicker {
```

Comment:
Sorry, I forgot the maps setting I pointed you to had this wrapper. IIRC that was just legacy, and I definitely don't think we should replicate it for this; instead clients would do lines 374-377 themselves. We would just document that in the Android package's README, and link to it in the main package's README.